KubeArmor/config/config.go

```diff
@@ -113,6 +114,9 @@ const ConfigCoverageTest string = "coverageTest"
 // ConfigK8sEnv VM key
 const ConfigK8sEnv string = "k8s"
 
+// Untracked Namespaces
+const UntrackedNs string = "untrackedNs"
```

Comment:
```suggestion
// Config Constants
const (
      ...
      ConfigUntrackedNs string = "untrackedNs"
      ...
 )
 
```

We have redundant comments here for exported constants, let's group them into a single constant declaration, this will prevent future lint warnings as well.
Let's use Config prefix for these constants, we are usually following it except LsmOrder and bpffs path.
For ref check
https://github.com/kubearmor/KubeArmor/blob/c81427eda39a1879206791a1504495ab37024b3a/KubeArmor/enforcer/bpflsm/rulesHandling.go#L16